### PR TITLE
possible fix for auto/unknown virus scanner

### PR DESCRIPTION
**Pull request by Skywalker-11**
This should prevent error messages like #463 where an unknown virus scanner is defined or 'auto'.
It will show an info message in the report table in the top right part of the page (not shown to normal users)

### CODE REVIEW

**Comment by stefaweb** [comment]
> 'unknownvirusscanner03' => 'Unknown virus scanner',

'unknownvirusscanner03' => 'Logiciel anti virus inconnu',

File: mailscanner/languages/en.php [hunk 1]
```diff
@@ -185,6 +185,7 @@
     '15minutes03' => '15 min.:',
     'saspam03' => 'Spam',
     'sanotspam03' => 'Not spam',
+    'unknownvirusscanner03' => 'Unknown virus scanner',
```
Comment (endelwar) [inline mailscanner/languages/en.php]: This should be more specific, like: 
Unknown virus scanner defined in MailScanner.conf, review your configuration and don't use 'auto' as antivirus name (see FAQ)

**Comment by asuweb** [comment]
@Skywalker-11 - does this also fix the issue with esets being the first virus scanner defined?

**Comment by stefaweb** [comment]
For fr.php:

`'unknownvirusscanner03' => 'Le scanner de virus défini dans MailScanner.conf est inconnu. Modifiez votre configuration et n\'utilisez pas \'auto\' comme nom d\'antivirus (voir FAQ).',`

**Comment by Skywalker-11** [comment]
@asuweb This should prevent broken pages but mailwatch still only uses the first virus scanner set in mailscanner.conf. So yes it should prevent the PHP Notice and PHP Warning with esets but can't use the virus data.

**Comment by asuweb** [comment]
@Skywalker-11 - great :)

**Comment by stefaweb** [comment]
Localization in #492 but DE needed to do for 'unknownvirusscanner03'.

**Pull request merged**